test_project/test_app/tests/test_notifications.py

```diff
@@ -224,3 +226,12 @@ def test_settings_only_updated_or_listed(self):
         self.assertSchemaPost(create_url, "$notificationSettingRequest", "$notificationSettingResponse", data, user,
                               unauthorized=True)
         self.assertSchemaDelete(delete_url, user, unauthorized=True)
+
+    def test_notification_types_are_cached(self):
+        cache.set('test', 'testval')
+        if cache.get('test'):  # If memcache is running
```

Comment:
I was thinking about using something like the following to be a bit cleaner about skipping: https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures